.travis.yml

```diff
@@ -1,5 +1,17 @@
 language: dart
-dart: dev
-script: ./tool/travis.sh
-dist: trusty
 sudo: false
+dart:
+  - dev
+  - stable
+dart_task:
+  - test
+  - dartfmt
```

Comment:
using dartfmt when also specifying dev and stable channels can get the CI into issues (when dartfmt changes)

Reply:
Yup...if/when.

It's easy to flip if a change emerges...